src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -50,9 +50,8 @@ def handle_exception(ex):
     # Otherwise, unhandled exception. Direct users to create an issue.
     logger.error("The command failed with an unexpected error. Here is the traceback:\n")
     logger.exception(ex)
-    logger.error("\nIf you think this is a bug, please visit: https://github.com/Azure/azure-cli to create an issue."
-                 "\nIf you would like tips for using the CLI, "
-                 "visit: https://github.com/Azure/azure-cli/blob/dev/doc/use_cli_effectively.md.")
+    logger.warning("\nTo open an issue, please visit: https://aka.ms/azure-cli-issues"
+                   "\nIf this command is an extension, please visit: https://aka.ms/azure-cli-extensions-issues")
```

Comment:
So, now my question is... we should know whether the command is an extension or not. It's how we know to display the "this command's behavior is overridden by blah extension" message. We should investigate whether we can access this metadata and simply provide the single correct link.

Reply:
Done!

![Issue link image.](https://user-images.githubusercontent.com/15149427/53930589-b03a2600-4046-11e9-8688-4b326a3ad029.png)

Reply:
Awesome!